### PR TITLE
[EDU-176] Migrate content to relevant sections and remove best practice guide

### PR DESCRIPTION
**Pull request by m-hulbert**
## Description

This PR removes the best practice guide and moves all content that hasn't yet been migrated, into relevant sections. 

It also addresses the documentation concerns raised in https://github.com/ably/docs/issues/2073.

An associated website PR will resolve the redirect.

### CODE REVIEW

File: content/presence-occupancy/presence.textile [hunk 1]
```diff
@@ -445,7 +446,7 @@ h2(#retrieve-members). Retrieve presence members
...
 
-An Ably client connected using the realtime interface of an SDK is responsible for keeping track of the presence set from the time that the channel is attached. An up to date presence set is pushed to the client following a channel attachment, and the presence set is updated on each subsequent presence event. This means that "get()":/api/realtime-sdk/presence#get returns the already known presence set retained in memory and does not trigger a new request to the Ably service. 
+An Ably client connected using the realtime interface of an SDK is responsible for keeping track of the presence set from the time that the channel is attached. An up to date presence set is pushed to the client following a channel attachment, and the presence set is updated on each subsequent presence event. This means that "get()":/api/realtime-sdk/presence#get returns the already known presence set retained in memory and does not trigger a new request to the Ably service.
```
Comment (franrob-projects) [inline content/presence-occupancy/presence.textile:449]: get() --> `get()` + link

File: content/presence-occupancy/presence.textile [hunk 2]
```diff
@@ -590,6 +591,28 @@ if page.hasNext() {
 }
 ```
 
+h3(#synced). Synced presence set
+
+A common need is to keep an updated list of members that are currently present on a channel in your user interface.
+
+Many developers try to build the initial list using the "@get()@":/api/realtime-sdk/presence#get method and then mutate that list whenever a new presence event arrives. This approach is not recommended by Ably as it's easy to get wrong and end up with a list that's out of sync with the real presence set.
```
Comment (franrob-projects) [inline content/presence-occupancy/presence.textile:598]: Possible reword for clarity? 

Ably advises against this method due to its high risk of errors, which could lead to discrepancies between your list and the actual presence set.

**Review by franrob-projects — APPROVED** [review]
I've only made minor suggestions!


Note. The enhancements made here also address the issues raised in https://github.com/ably/docs/issues/2073.

File: content/channels/index.textile [hunk 3]
```diff
@@ -667,6 +671,8 @@ h2(#subscribe). Subscribe to a channel
 
 Subscribe to a channel in order to receive messages being published to it, by registering a listener. Subscribing is an operation available to the realtime interface and uses the "@subscribe()@":/api/realtime-sdk/channels#subscribe method.
 
+Subscribing to events server-side using the Pub/Sub method can be disadvantageous as it can increase latency or duplicate events among multiple servers. "Message Queues":/general/queues are more a appropriate method to use in that instance, as multiple worker servers enable Ably to distribute the load of messages received from published. This means that each message is only processed once by any one of your worker servers.
```
[the remark this answers is not in the document]
Comment (GregHolmes) [reply content/channels/index.textile:674]: I'm not sure. I think it reads right. They're not events specifically for the server-side, but it's the server-side that's subscribing to the events.

File: content/channels/index.textile [hunk 4]
```diff
@@ -1453,6 +1459,8 @@ Previously registered listeners can be removed individually or all together.
     stateChangeListener.cancel();
 ```
 
+Be aware that when registering listeners for channel state changes certain repeating states may add new listeners each time.
```
[the remark this answers is not in the document]
Comment (m-hulbert) [reply content/channels/index.textile]: Updated, thanks!

**Pull request merged**